test/structuredbroadcast.jl

```diff
@@ -388,4 +393,11 @@ end
     @test ind == CartesianIndex(1,1)
 end
 
+@testset "Rectangular UpperHessenberg" begin
+    UH = UpperHessenberg(ones(4,3))
+    UH2 = UH .+ UH .- UH
```

Comment:
Would that even work with

```suggestion
    UH2 = UH + UH - UH
```

?

Reply:
Yes, it would. Should I add a test to compare these?

Reply:
Yes, it would. Should I add a test to compare these?